Makefile

```diff
@@ -51,7 +51,6 @@ response_parser.o: response_parser.h response_parser.cpp
 .PHONY: clean, all, test, deploy
 
 deploy:
-	make
```

Comment:
this isnt a problem for us because docker always builds from source. Inspect the Dockerfile in our main directory and you'll notice that after it installs the necessary dependencies such as boost and g++, it runs make. Running make before deploy.sh is redundant in this case.